GameData/RP-0/Strategies/Leaders/LeadersEngineers.cfg

```diff
@@ -389,3 +389,59 @@ STRATEGY
 		}
 	}
 }
+
+STRATEGY
+{
+	name = leaderValCleaverEngineer
+	RP0conf = True
+	title = Arthur Valentine Cleaver
+	desc = Arthur "Val Cleaver" is a British Aerospace Engineer working for de Havilland in the early 1950s, responsible for overseeing the development of the early British rocket plane engines, "Spectre" and "Sprite". Later he worked to develop upgraded variants of the "Spectre" engine, that went on to power the Blue Steel missile.  In 1956, Val Cleaver hands in his notice to de Havilland, and takes a job with Rolls-Royce as their Chief Rocket Propulsion Engineer. Under his guidance the development of the RZ.2 and RZ.20 takes place, built to power the Blue Streak missile. Considered highly advanced engines at the time, it later goes on to power the first stage of Europa 1. Arthur Val Cleaver is awarded an Order of the British Empire, a prestigious medal for prominent national achievements.
+	department = Engineering
+	icon = RP-0/Strategies/Leaders/ValCleaver
+	iconDepartment = RP-0/Strategies/Leaders/ValCleaver
+
+
+	groupTag = Engineering,leaderValCleaver
+
+	removeOnDeactivate = True
+	removeOnDeactivateTag = leaderValCleaver
+  
+	minDuration = 2592000 // 30 days
+	maxDuration = 315576000 // 10 years
+	reactivateCooldown = 31557600 // 1 year
+	
+	EFFECT
+	{
+		name = ResearchRateModifier
+		effectTitle = Rolls-RoyceDidRocketsGood!
+		multiplier = 1.05
+		effectDescription = Early and Orbital Rocket Engine tech
+		nodeTypes
+		{
+			typeNode = RocketEngines
+		}
+	}
+
+	EFFECT
+	{
+		name = ResearchRateModifier
+		effectTitle = BritishRocketPlanes
+		multiplier = 1.05
+		effectDescription = Flight tech
+		nodeTypes
+		{
+			typeNode = Flight
+		}
+	}
+
+	EFFECT
+	{
+		name = ResearchRateModifier
+		effectTitle = RZ.20 never took off
+		multiplier = 0.9
+		effectDescription = Flight tech
+		nodeTypes
+		{
+			typeNode = Hydrolox
```

Comment:
As above re: hydrolox penalties, it feels like Pap made half the leaders have those!

Reply:
re above yourself!:), its just a British guy doing the same thing as everyone else. I Can think an alternative negative (if one at All?) being tankage, Blue Streak was a corrogated stainless-steel conventional tank (https://www.globalsecurity.org/wmd/world/uk/blue-streak-design.htm) if thats more prefferable!

Reply:
For some reason I recalled Blue Streak being balloon, with some stiffening stringers on the outside of the after tank?

Reply:
Might well be, I dont know the legitimacy of the source, what caught my eye was "The skin was supported on a riveted structure of internal frames and stringers" and "~was further stiffened by spotwelding longitudinal top-hat stringers around its outer circumference." though the thickness of the sheets is what you would see with an Atlas Balloon tank, a bit of a mashup of sorts perhaps

Reply:
_A Vertical Empire_ might have the fullest details?

Reply:
I think you're right, seems a good source, it reads there were only external stringers on the rear tank that would support the weight of the front tank to be supported without pressuring the rear tank. and the upper tank Was pressurized